src/utils/browserPaths.ts

```diff
@@ -35,7 +35,10 @@ export const hostPlatform = ((): BrowserPlatform => {
     const macVersion = execSync('sw_vers -productVersion', {
       stdio: ['ignore', 'pipe', 'ignore']
     }).toString('utf8').trim().split('.').slice(0, 2).join('.');
-    const archSuffix = os.arch() === 'arm64' ? '-arm64' : '';
+    const arm64 = execSync('sysctl -in hw.optional.arm64', {
```

Comment:
Let's restrain this method to macOS 11+ and Node <15 ?

Reply:
Node 15 reports 'x64' unless you compile it yourself.